### PR TITLE
[add] updated ts.info with the latest reply format

### PR DESCRIPTION
**Pull request by filipecosta90**
[add] updated ts.info with the latest reply format

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -358,33 +358,64 @@ TS.MGET FILTER filter...
...
+#### Complexity
+
+`TS.INFO` complexity is O(1).
+
+#### Return Value
+
+Array-reply, specifically:
+
+- Total samples in the Serie.
+- Total number of bytes allocated for the Series.
```
Comment (K-Jo) [inline docs/commands.md]: consistent use of serie  / series

File: docs/commands.md [hunk 2]
```diff
@@ -358,33 +358,64 @@ TS.MGET FILTER filter...
...
+- First timestamp present in the Serie.
+- Last timestamp present in the Serie.
+- Retention time, in milliseconds, for the Series.
```
Comment (K-Jo) [inline docs/commands.md]: same

File: docs/commands.md [hunk 3]
```diff
@@ -358,33 +358,64 @@ TS.MGET FILTER filter...
...
+- Number of Memory Chunks used for the Series.
```
Comment (K-Jo) [inline docs/commands.md]: same

File: docs/commands.md [hunk 4]
```diff
@@ -358,33 +358,64 @@ TS.MGET FILTER filter...
...
+Returns information and statistics on the series. 
+
+#### Parameters
+
 * key - Key name for timeseries
 
-#### Info Example
+#### Complexity
+
+`TS.INFO` complexity is O(1).
```
Comment (K-Jo) [inline docs/commands.md]: Is this in line with other?  I thought we just said 'O(1)' without the sentence

Comment (K-Jo) [reply docs/commands.md]: It's not, but just put O(1) will revise the others

**Comment by filipecosta90** [comment]
@K-Jo all changes are in place right? :)

**Pull request merged**